### PR TITLE
CLI Enhancements

### PR DESCRIPTION
**Pull request by theweaklink**
Add convenient options to CLI commands:
- `onecode-start`: option `--dump` allows to only generate the `app.py` Streamlit file without restarting the web server => useful to keep the webserver opened and regenerate the file on the fly
- `onecode-start` and `onecode-extract`: option `--verbose` is useful for printing out all functions being processed (and potentially debug when an element is not processed), e.g.
```
Processing "Step1"...
 >> (flows.step1.run) function onecode.csv_reader ✅
 >> (flows.step1.run) function onecode.dropdown ✅
 >> (flows.step1.run) function onecode.Logger.info 🚫
 >> (flows.step1.run) function onecode.Logger.info 🚫
 >> (flows.step1.run) function <builtin>.range 🚫
 >> (flows.step1.run) function time.sleep 🚫
 >> (flows.step1.run) function onecode.Logger.info 🚫
 >> (flows.step1.run) function onecode.csv_output ✅
 >> (flows.step1.run) function onecode.Logger.info 🚫
Processing "Step2"...
 >> (flows.step2.run) function onecode.Project 🚫
 >> (flows.step2.run) function onecode.Logger.info 🚫
 >> (flows.step2.run) function onecode.Logger.info 🚫
 >> (flows.step2.run) function onecode.Logger.info 🚫
 >> (flows.step2.run) function onecode.Logger.warning 🚫
 >> (flows.step2.run) function onecode.Logger.error 🚫
 >> (flows.step2.run) function onecode.Logger.critical 🚫
 >> (flows.step2.run) function flows.utils.xx 🚫
Processing "Step3"...
 >> (flows.step3.run) function onecode.slider ✅
 >> (flows.step3.run) function onecode.Logger.info 🚫
 >> (flows.step3.run) function onecode.file_input ✅
 >> (flows.step3.run) function onecode.Logger.info 🚫
 >> (flows.step3.run) function onecode.file_input ✅
 >> (flows.step3.run) function onecode.Logger.info 🚫
```

### CODE REVIEW

**Review by hameye — CHANGES_REQUESTED** [review]
2 minor comments
Instead of the prohibited symbol (https://emojipedia.org/prohibited/) what about using the fast forward symbol ? (https://emojipedia.org/fast-forward-button/) ?

File: onecode/cli/extract.py [hunk 1]
```diff
@@ -116,9 +117,9 @@ def main(cli: bool = True) -> None:    # pragma: no cover
     """
     ```bash
     usage: onecode-extract [-h] [--all] [--modules [MODULES [MODULES ...]]] [--path PATH]
-        output_file
+        [--verbose] output_file
 
-    Extract OneCode project parameters to JSON file.
+    Extract OneCode project parameters to JSON file
```
[the remark this answers is not in the document]
Comment (theweaklink) [reply onecode/cli/extract.py:122]: there was no "dot" when running `onecode-extract --help` so I removed it to be consistent with the CLI help

File: onecode/cli/start.py [hunk 2]
```diff
@@ -447,6 +450,16 @@ def main() -> None:   # pragma: no cover
...
+        '--dump',
+        action="store_true",
+        help='Only generate the app.py file'
+    )
+    parser.add_argument(
+        '--verbose',
+        help='Print verbose information when processing files',
+        action='store_true'
+    )
     args = parser.parse_args()
```
[the remark this answers is not in the document]
Comment (theweaklink) [reply onecode/cli/start.py:463]: yes indeed, but it is already the same order no? dump then verbose? or am I missing something?
(same as above, I copy paste the content of `onecode-start --help` in the doc

**Comment by theweaklink** [comment]
Agreed on the skip symbol, thanks!

File: onecode/cli/start.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (hameye) [reply onecode/cli/start.py:463]: @theweaklink sorry for being unclear, i was referring to action/help vs help/action

**Pull request merged**